api/objects.proto

```diff
@@ -249,6 +249,14 @@ message Cluster {
 	// If the key is empty, the node will be unlocked (will not require a key
 	// to start up from a shut down state).
 	repeated EncryptionKey unlock_keys = 9;
+	
+	// RoutingMeshOff denotes whether the routing mesh is currently disabled.
+	// By default the routing mesh is enabled. As result of user removing the
+	// ingress network, this flag will be set to true. Cluster needs to carry
+	// this information, so that on reboot or leader re-election, new leader
+	// will not automatically create the routing mesh if it cannot find one
+	// in store. 
+	bool routingMeshOff = 10;
```

Comment:
A few comments here:

- protobuf names should use underscores, for example `routing_mesh_off`. This gets converted to CamelCase by the protobuf compiler for Go code.
- Maybe `routing_mesh_disabled` is better?
- It's too bad we don't have a message inside `Cluster` to store networking options, to organize this and stuff like `network_bootstrap_keys` and `encryption_key_lamport_clock`. Maybe it's not too late to start one, though.